sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -840,6 +989,100 @@ def simple_udf(x: int):
     test_registry.teardown()
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "test_registry",
+    all_fixtures,
+)
+def test_apply_feature_service_success(test_registry):
+    # Create Feature Service
+    file_source = FileSource(name="my_file_source", path="test.parquet")
+    feature_view = FeatureView(
+        name="my_feature_view",
+        entities=[],
+        schema=[
+            Field(name="feature1", dtype=Float32),
+            Field(name="feature2", dtype=Float32),
+        ],
+        source=file_source,
+    )
+    fs = FeatureService(
+        name="my_feature_service_1", features=[feature_view[["feature1", "feature2"]]]
+    )
+    project = "project"
+
+    # Register Feature Service
+    test_registry.apply_feature_service(fs, project)
+
+    feature_services = test_registry.list_feature_services(project)
+
+    # List Feature Services
+    assert len(feature_services) == 1
+    assert feature_services[0] == fs
+
+    # Delete Feature Service
+    test_registry.delete_feature_service("my_feature_service_1", project)
+    feature_services = test_registry.list_feature_services(project)
+    assert len(feature_services) == 0
+
+    test_registry.teardown()
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "test_registry",
+    all_fixtures,
+)
+def test_modify_feature_service_success(test_registry):
+    # Create Feature Service
+    file_source = FileSource(name="my_file_source", path="test.parquet")
+    feature_view = FeatureView(
+        name="my_feature_view",
+        entities=[],
+        schema=[
+            Field(name="feature1", dtype=Float32),
+            Field(name="feature2", dtype=Float32),
+        ],
+        source=file_source,
+    )
+    fs = FeatureService(
+        name="my_feature_service_1", features=[feature_view[["feature1", "feature2"]]]
+    )
+    project = "project"
+
+    # Register Feature service
+    test_registry.apply_feature_service(fs, project)
+
+    feature_services = test_registry.list_feature_services(project)
+
+    # List Feature Services
+    assert len(feature_services) == 1
+    assert feature_services[0] == fs
+
+    test_registry.delete_feature_service("my_feature_service_1", project)
```

Comment:
Should you be doing this? If you delete and apply again, of course `created_timestamp` will be regenerated. Isn't the whole point to assert that it's unchanged even if new object is applied when during apply an object already exists?

Reply:
My bad, this was supposed to be present for `test_apply_feature_service_success` only. I'll remove this.